app/models/author.rb

```diff
@@ -0,0 +1,9 @@
+class Author < ApplicationRecord
+  #association
+  has_many :articles # An author can have many articles.
+
+  #validations
+  validates :name, presence: true
```

Comment:
add length validation and custom message

Reply:
Changes implemented, please check latest commit.